src/consensus/params.h

```diff
@@ -151,8 +151,10 @@ struct Params {
     /** Block height at which DIP0001 becomes active */
     int DIP0001Height;
     /** Block height at which DIP0003 becomes active */
-    int DIP0003Height;
-    uint256 DIP0003Hash;
+    int DIP0003ActivationHeight;
```

Comment:
This seems to be unused?

Reply:
Ooops, this one is slipped through from the future (code changes) :D